src/Pages/JobDetails/JobDetails.js

```diff
@@ -0,0 +1,236 @@
+import React, { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import styled from 'styled-components';
+import Slide from './Components/Slide';
+import Map from './Components/Map';
+import ApplicationBox from './Components/ApplicationBox';
+import AsideButton from './Components/AsideButton';
+import {
+  JOB_DETAILS_API,
+  JOB_DETAILS_USER_API,
+  JOB_DETAILS_RESUME_API,
+} from '../../config';
+
+const JobDetails = () => {
+  const [isAsideOpen, setIsAsideOpen] = useState(true);
+  const [details, setDetails] = useState([]);
+  const [userInfo, setUserInfo] = useState([]);
+  const [fileList, setFileList] = useState([]);
+  const [formList, setFormList] = useState([]);
+  const params = useParams();
+  // const reuseFetch = (api, token) => {
+  // 	fetch(api, {
+  //     headers: {
+  //       authorization: 'token',
+  //     },
+  //   }).then(res=>res.json())
+  // }
+
+  const handleAside = value => {
+    if (value === '지원하기') {
+      fetch(`${JOB_DETAILS_USER_API}=${params.id}`, {
```

Comment:
- config.js 부분에 리뷰 남겼는데, API 부에 쿼리스트링 **일부분** 포함시키지 말아주세요!
- `=123` 와 같이 나오게 되는데, 이렇게 되면 이것이 쿼리스트링인지 다른 무엇인지 바로 파악하기 어렵습니다.
```suggestion
fetch(`${JOB_DETAIL_USER_API}/?notification=${params.id}`)
```

Reply:
넵 수정하였습니당!